site/en/r2/tutorials/generative/dcgan.ipynb

```diff
@@ -1,143 +1,197 @@
 {
```

Comment:
<p>This chunk would be easier to read if we `import tensorflow.keras.layers as layers`, then `model.add(layers.Dense(...))`</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Agreed &amp; updated, thanks.</p>